substra/sdk/schemas.py

```diff
@@ -227,17 +226,17 @@ class ComputePlanSpec(_BaseComputePlanSpec):
     """
 
     key: str = pydantic.Field(default_factory=lambda: str(uuid.uuid4()))
-    tag: Optional[str]
+    tag: Optional[str] = None
     name: str
-    metadata: Optional[Dict[str, str]]
+    metadata: Optional[Dict[str, str]] = None
 
     type_: typing.ClassVar[Type] = Type.ComputePlan
 
     @contextlib.contextmanager
     def build_request_kwargs(self):
         # default values are not dumped when `exclude_unset` flag is enabled,
         # this is why we need to reimplement this custom function.
-        data = json.loads(self.json(exclude_unset=True))
+        data = json.loads(self.model_dump_json(exclude_unset=True))
```

Comment:
is this workaround still the way to go with the latest version?

Reply:
Didn't look into another implementation, but your right there should be a better way...